rundeck/manifest.json

```
@@ -0,0 +1,31 @@
+{
+  "display_name": "Rundeck",
+  "maintainer": "forrest@rundeck.com",
+  "manifest_version": "1.0.0",
+  "name": "Rundeck",
+  "metric_prefix": "rundeck.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Automate Remediation Actions using Rundeck Webhooks",
+  "guid": "2e3492d3-14fd-409d-b339-fb1bc14d7db9",
+  "support": "core",
```

```suggestion
  "support": "contrib",
```
CI is failing manifest validation